developer-extension/src/panel/components/tabs/eventsTab/eventsList.module.css

```diff
@@ -1,7 +1,23 @@
 .root {
   margin: 0 var(--mantine-spacing-md) var(--mantine-spacing-md) var(--mantine-spacing-md);
-  border: 1px solid var(--dd-border-color);
-  border-radius: var(--mantine-radius-md);
+}
+
+.root thead {
+  z-index: var(--dd-events-list-header-z-index);
+}
+
+/*
```

Comment:
**question:**
have you considered using a `box-shadow` for your sticky table header?
you can get some inspiration from what is done in druids Header component, [here](https://github.com/DataDog/web-ui/blob/aaa6207a8c6e93b1c093875e32fa9892dee79dc9/packages-to-migrate/druids-ui/layout/Header/header.less)'s the stylesheet